### PR TITLE
Update for Antistasi 3.0

### PR DESCRIPTION
**Pull request by targetingsnake**
Updated documentation with the information from the attack support rework PR.

### CODE REVIEW

File: source/reference_guide/concepts_info/concepts/AgressionUpdateLoop.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+   .. rst-class:: code-paragraph
+
+      - :code:`A3A_activePlayerCount` is a count of all active players. Activity is determined by a simple client-side getDir check.
+      - :code:`A3A_balancePlayerScale` is the general amount-of-stuff coefficient, normalized to 1 with 6 players at tier 6.
+      - :code:`A3A_balanceVehicleCost` is a rough average resource cost of a single attack vehicle + cargo (if any) at the current war tier.
+      - :code:`A3A_balanceResourceRate` is the resources of each type typically gained by a faction every 10 minutes, currently just :code:`balancePlayerScale * balanceVehicleCost`. The defence cap is currently at :code:`10 * A3A_balanceResourceRate`.
+      - :code:`A3A_enemyBalanceMul` is a parameter (default 10, because params don't do FP) that multiplies :code:`A3A_balancePlayerScale` to make the game generally harder or easier.
+      - :code:`A3A_enemyAttackMul` is a parameter that multiplies attack resource rate to make attacks faster or slower.
+      - :code:`A3A_invaderAttackMul` and :code:`A3A_invaderDefenceMul` are temporary multipliers applied to invader resource gain rates. These aren't currently parameterized.
```
Comment (jaj22) [inline source/reference_guide/concepts_info/concepts/AgressionUpdateLoop.rst:23]: Changed. These two were replaced by A3A_invaderBalanceMul, which is parameterized.

File: source/reference_guide/concepts_info/concepts/attack_choice_system.rst [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+   Description
+   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+   Once a faction has positive attack resources, it will call chooseAttack to launch an attack. Target selection is handled with findAttackTargets, which weights potential attacks depending on target value and threat. Main differences:
+
+   - Lower-rated attacks can be chosen, using the weight as a probability.
+   - Attacks against rebel targets outside mission distance are now permitted, although chance is reduced.
+   - Danger of flying over other enemy locations is taken into account.
+   - Threat of other nearby locations is taken into account, not just the target.
+   - Distance has no effect (other than the hard air distance cap), but nearby land bases are beneficial.
```
Comment (jaj22) [inline source/reference_guide/concepts_info/concepts/attack_choice_system.rst:19]: Changed, air distance now has some effect, although less than previously.

File: source/reference_guide/concepts_info/concepts/hq_attack.rst [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+.. card::
+   :class-card: sd-mt-3
+   :class-header: header-2-light
+
+   Description
+   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+   HQ attacks can now only be launched once an enemy faction has full knowledge of your HQ. Knowledge is currently added when a traitor mission is failed, or when a support is called against a target near HQ. Knowledge diminishes slowly over time as long as it hasn't reached 1. Moving HQ will reset knowledge, as long as their wasn't a previous HQ nearby recently.
+
+   The attacks themselves are still specops air, but benefit from improvements to combat behaviour. Support artillery and UAVs are also used to assist.
```
Comment (jaj22) [inline source/reference_guide/concepts_info/concepts/hq_attack.rst:15]: Changed, HQ attacks can now include ground components, although less than a major attack.

File: source/reference_guide/concepts_info/concepts/resourcePools.rst [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+   Enemy resources are split into two types, "attack" and "defence".
+   Units generated as part of an attack or support are paid for up-front and have their "A3A_resPool" variable set accordingly.
+   If despawned in good condition, they are partially refunded. Mission units have A3A_resPool set to "legacy" and are free
+   to spawn (as they're effectively ambient units with no particular task), and paid for equally from both pools on destruction.
+   Garrison units are paid for on creation and not refunded on despawn for obvious reasons. Attack and defence resources are saved,
+   and updated appropriately on save: Pre-paid units are refunded according to damage level.
+
+   Attack and defence pools are used differently. An attack is launched once an enemy faction hits >0 attack resources,
+   typically leaving that faction with a large negative value, which is slowly increased back to zero before the next attack
+   can be launched. Defence resources typically have a positive buffer which is used up as reinforcements and supports are generated.
```
Comment (jaj22) [inline source/reference_guide/concepts_info/concepts/resourcePools.rst:22]: Bad line breaks in this one.

File: source/reference_guide/concepts_info/concepts/supportSystem.rst [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+   - createSupport is a worker function to force-create a support of a specific type. It will use already-active supports by preference, if they don't currently have a target.
+
+   Each support typically has an availability function (actually returns a weight now, could be renamed), a creation function (intended to be called by createSupport, and so returns fairly quickly), and a persistently spawned routine that manages the support while active. Multi-target supports update the A3A_activeSupports array to indicate readiness and completion.
+
+   All working supports from the previous system have been preserved except for Gunship and the orbital laser, which need updating to new systems.
```
Comment (jaj22) [inline source/reference_guide/concepts_info/concepts/supportSystem.rst:22]: Changed, orbital laser is now working.

File: source/reference_guide/concepts_info/concepts/hq_attack.rst (hunk 3 above)
Comment (targetingsnake) [reply source/reference_guide/concepts_info/concepts/hq_attack.rst:15]: i tried to put that into the sentence please check after i commited the changes (will ping you)

File: source/reference_guide/concepts_info/concepts/resourcePools.rst (hunk 4 above)
Comment (targetingsnake) [reply source/reference_guide/concepts_info/concepts/resourcePools.rst:22]: line breaks will be done by browser, only line breaks with a blank line inbetween matters

File: source/reference_guide/concepts_info/concepts/supportSystem.rst (hunk 5 above)
Comment (targetingsnake) [reply source/reference_guide/concepts_info/concepts/supportSystem.rst:22]: removed orbital laser from that line

File: source/reference_guide/concepts_info/concepts/attack_choice_system.rst (hunk 2 above)
Comment (targetingsnake) [reply source/reference_guide/concepts_info/concepts/attack_choice_system.rst:19]: can you tell me which impact or rephrase that more friendly for people? i can't follow what you wanna tell me with that

File: source/reference_guide/concepts_info/concepts/resourcePools.rst (hunk 4 above)
Comment (targetingsnake) [reply source/reference_guide/concepts_info/concepts/resourcePools.rst:22]: looks in the end like this 
image

File: source/reference_guide/concepts_info/concepts/AgressionUpdateLoop.rst [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+   .. rst-class:: code-paragraph
+
+      - :code:`A3A_activePlayerCount` is a count of all active players. Activity is determined by a simple client-side getDir check.
+      - :code:`A3A_balancePlayerScale` is the general amount-of-stuff coefficient, normalized to 1 with 6 players at tier 6.
+      - :code:`A3A_balanceVehicleCost` is a rough average resource cost of a single attack vehicle + cargo (if any) at the current war tier.
+      - :code:`A3A_balanceResourceRate` is the resources of each type typically gained by a faction every 10 minutes, currently just :code:`balancePlayerScale * balanceVehicleCost`. The defence cap is currently at :code:`10 * A3A_balanceResourceRate`.
+      - :code:`A3A_enemyBalanceMul` is a parameter (default 10, because params don't do FP) that multiplies :code:`A3A_balancePlayerScale` to make the game generally harder or easier.
+      - :code:`A3A_enemyAttackMul` is a parameter that multiplies attack resource rate to make attacks faster or slower.
+      - :code:`A3A_invaderBalanceMul` are temporary multipliers applied to invader resource gain rates. These aren't currently parameterized.
```
[the remark this answers is not in the document]
Comment (jaj22) [reply source/reference_guide/concepts_info/concepts/AgressionUpdateLoop.rst:23]: Ok, this language doesn't even work with github markdown.

[the remark this answers is not in the document]
Comment (targetingsnake) [reply source/reference_guide/concepts_info/concepts/AgressionUpdateLoop.rst]: changed thx for the rephrasing/rewording

File: source/reference_guide/concepts_info/concepts/attack_choice_system.rst (hunk 2 above)
Comment (jaj22) [reply source/reference_guide/concepts_info/concepts/attack_choice_system.rst:19]: Distance from air bases has less effect, with nearby land bases being more important.

File: source/reference_guide/concepts_info/concepts/supportSystem.rst [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+   - createSupport is a worker function to force-create a support of a specific type. It will use already-active supports by preference, if they don't currently have a target.
+
+   Each support typically has an availability function (actually returns a weight now, could be renamed), a creation function (intended to be called by createSupport, and so returns fairly quickly), and a persistently spawned routine that manages the support while active. Multi-target supports update the A3A_activeSupports array to indicate readiness and completion.
+
+   All working supports from the previous system have been preserved except for Gunship, which need updating to new systems.
```
Comment (jaj22) [inline source/reference_guide/concepts_info/concepts/supportSystem.rst:22]: need -> needs now.

Comment (targetingsnake) [reply source/reference_guide/concepts_info/concepts/supportSystem.rst]: changed

File: source/reference_guide/concepts_info/concepts/AgressionUpdateLoop.rst (hunk 1 above)
Comment (targetingsnake) [reply source/reference_guide/concepts_info/concepts/AgressionUpdateLoop.rst]: changed

File: source/reference_guide/concepts_info/concepts/attack_choice_system.rst (hunk 2 above)
Comment (targetingsnake) [reply source/reference_guide/concepts_info/concepts/attack_choice_system.rst]: replaced sentence

**Pull request merged**